scripts/portal-down.sh

```diff
@@ -2,15 +2,17 @@
 
 set -e # exit on first error
 
-while getopts d:t: flag
+while getopts d:t:r: flag
 do
     case "${flag}" in
         d) delay=${OPTARG};;
         t) timeout=${OPTARG};;
+        r) reason=${OPTARG};;
     esac
 done
 delay=${delay:-0} # default to no delay
 timeout=${timeout:-300} # default timeout is 300s
+reason=${reason:-"disabled manually"}
```

Comment:
I think we shouldn't provide default reason, if none is provided cli will ask for a reason